example_test.go

```diff
@@ -67,3 +64,44 @@ func ExampleWithProcessRunner_logging() {
 		panic(err)
 	}
 }
+
+func ExampleJailer_withBuilder() {
```

Comment:
Yea, goal here was how configurable the jailer builder is.

Reply:
It'd be good to have an example that is the simplest working way as well (unless that's already covered by another example because the jailer is used by default?).